### PR TITLE
Override node inputs

### PR DESCRIPTION
**Pull request by chenslucky**
# Description

(https://ml.azure.com/runs/great_house_1c5cvbzcsq?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourcegroups/promptflow/workspaces/yaopfeus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)

https://ml.azure.com/runs/loving_bean_2q07s6qs6x?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourcegroups/promptflow/workspaces/yaopfeus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/gen_test_data/gen_test_data/run.py [hunk 1]
```diff
@@ -252,58 +253,65 @@ def get_ml_client(subscription_id: str, resource_group: str, workspace_name: str
...
+                "Skip step 1 'Split documents to document nodes' as received document nodes from "
+                f"input file '{args.document_nodes_file}'."
+            )
+            logger.info(f"Collected {count_non_blank_lines(args.document_nodes_file)} document nodes.")
+
+        copy_flow_folder_and_set_node_inputs(copied_flow_folder, args.flow_folder, args.node_inputs_override)
+
+        if args.cloud:
+            run_cloud(
+                args.documents_folder,
```
Comment (16oeahr) [inline examples/gen_test_data/gen_test_data/run.py:289]: please check merge error

File: .gitignore [hunk 2]
```diff
@@ -171,6 +171,8 @@ connection.json
 .azureml
 # dummy custom tool package example
 hello-world-proj/**
+# temp gen test data folder
+examples/gen_test_data/gen_test_data/*_temp
```
Comment (16oeahr) [inline .gitignore:175]: should we ignore "gen_test_data/config.ini" ?

Comment (chenslucky) [reply .gitignore:175]: ignore

File: examples/gen_test_data/gen_test_data/run.py [hunk 3]
```diff
@@ -252,58 +253,65 @@ def get_ml_client(subscription_id: str, resource_group: str, workspace_name: str
         "--prs_allowed_failed_count", type=int, help="Number of failed mini batches that could be ignored"
     )
     args = parser.parse_args()
+    copied_flow_folder = args.flow_folder + "_" + time.strftime("%b-%d-%Y-%H-%M-%S") + "_temp"
```
Comment (chjinche) [inline examples/gen_test_data/gen_test_data/run.py:256]: move this line to line 284 before `copy_and_set`

**Review by chjinche — APPROVED** [review]

**Pull request merged**